bind/files/redhat/named.conf

```diff
@@ -32,6 +32,12 @@ options {
         managed-keys-directory "/var/named/dynamic";
 };
 
+{%- if salt['pillar.get']('bind:statistics', {}) is defined %}
+statistics-channels {
+	inet 127.0.0.1 port 8053;
```

Comment:
would love to see the ability to configure this through pillar

Reply:
Btw, it's already possible to configure this option, as well as other ones, through bind:config:statistics, so I have to admit that I don't really see a capital gain.